packages/kolme-store/src/error.rs

```diff
@@ -1,15 +1,36 @@
 use merkle_map::{MerkleSerialError, Sha256Hash};
 
-#[derive(thiserror::Error, Debug)]
+#[derive(Debug, Clone, Copy, serde::Serialize, serde::Deserialize)]
+pub enum StorageBackend {
+    Fjall,
+    Postgres,
+    InMemory,
+}
+
+impl std::fmt::Display for StorageBackend {
```

Comment:
It looks like Display could be derived rather than needing an explicit implementation.